staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -339,6 +369,25 @@ func (w *watchCache) WaitUntilFreshAndGet(resourceVersion uint64, key string, tr
 	return value, exists, w.resourceVersion, err
 }
 
+// WaitUntilFreshAndListWithIndex returns a pointers to <storeElement> object, try to list through the index.
+func (w *watchCache) WaitUntilFreshAndListWithIndex(resourceVersion uint64, matchValues []storage.MatchValue, trace *utiltrace.Trace) ([]interface{}, uint64, error) {
+	err := w.waitUntilFreshAndBlock(resourceVersion, trace)
+	defer w.RUnlock()
+	if err != nil {
+		return nil, 0, err
+	}
+
+	// trying to query by index sequentially
```

Comment:
Remove this comment - it doesn't provide any value.

Reply:
fix it!